README.md

```diff
@@ -231,24 +231,27 @@ These are not blockers for any of the core sequence and so they are worked into
 1. [Diversity, Inclusion & Implicit Bias](/diversity-inclusion-bias/inclusion.md)
 1. [Tech Tips and Rules](https://docs.google.com/document/d/1SWiUl0lYQ0sYDM8vQnVhJVOvT4MRA-9pDglKOgqTWak/edit)
 1. [Giving Presentations](giving-presentations/giving-presentations.md)
-1. [Whiteboarding](/career/interviewing/whiteboarding.md)
-1. [Whiteboarding Interview Challenges](/career/interviewing/week-3-whiteboard-challenges.md)
+1. [Whiteboarding](/interviewing/whiteboarding.md)
+1. [Whiteboarding Interview Challenges](/interviewing/week-3-whiteboard-challenges.md)
 1. [Agile Project Management](/agile-development/intro-to-agile.md)
 1. [Intro to Scrum](/agile-development/scrum.md)
 
 ### Optional / Elective Lessons
 
-- [SEO: Search Engine Optimization](/seo/seo.md)
-- [Sass](/sass/sass.md)
-- [Python](/python/python.md)
-- [Datetime](/datetime/datetime.md)
```

Comment:
Thank you for fixing these.  Can you please move this Datetime line to line 105?

Reply:
should i replace line 105 with datetime line or just add?

Reply:
just add